packages/addons/service/librespot/package.mk

```diff
@@ -0,0 +1,67 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#      Copyright (C) 2017 Shane Meagher (shanemeagher)
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="librespot"
+PKG_VERSION="28232b5"
+PKG_REV="100"
+PKG_ARCH="x86_64 arm"
+PKG_ADDON_PROJECTS="Generic RPi2"
+PKG_LICENSE="prop."
+PKG_SITE="https://github.com/plietar/$PKG_NAME/"
+PKG_URL="https://github.com/awiouy/librespot-binaries/archive/$PKG_VERSION.tar.gz"
+PKG_SOURCE_DIR="librespot-binaries-$PKG_VERSION"
+PKG_DEPENDS_TARGET="toolchain pyalsaaudio"
+PKG_SECTION="service"
+PKG_SHORTDESC="Librespot: use Spotify Connect through LibreELEC"
+PKG_LONGDESC="Librespot ($PKG_VERSION) plays Spotify through LibreELEC using the opensource librespot library and, using a Spotify app as a remote."
+PKG_AUTORECONF="no"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="Librespot"
+PKG_ADDON_TYPE="xbmc.service"
+
+make_target() {
+  :
+}
+
+makeinstall_target() {
+  :
+}
+
+addon() {
+  case $TARGET_ARCH in
+    x86_64)
+      bin="x86_64"
+      ;;
+    arm)
+      bin="arm"
+      ;;
+    aarch64)
+      bin="aarch64"
+      ;;
+  esac
+
+  mkdir -p "$ADDON_BUILD/$PKG_ADDON_ID/bin"
+  cp "$PKG_BUILD/$bin/librespot" \
```

Comment:
Why not just use `$TARGET_ARCH` in place of `$bin` and remove the `case` statement (lines 48-58) which is then redundant?